NString.Tests/StringTemplateConverterTests.cs

```diff
@@ -0,0 +1,335 @@
+﻿using System.Globalization;
+using System.Linq;
+using Xunit;
+// ReSharper disable InconsistentNaming
```

Comment:
Is this actually necessary? Most of the other tests don't have it (I don't have R# on this machine so I can't see if it changes anything)

Reply:
I'm not sure anymore and I can't check on my current machine either.
`StringTemplateTests` had this and it's the class I used as a reference for my tests.

I will remove it.